libbeat/outputs/elasticsearch/api_test.go

```diff
@@ -1,3 +1,5 @@
+// +build unit integration
```

Comment:
The problem is that unit and integration tests share some helper functions which are inside api_test. I Agree that we should put these functions into an independent file in the future (testing.go?), but right now it is the simplest solution.